source/cli.txt

```diff
@@ -20,9 +20,9 @@ Overview
 .. binary:: realm-cli
 
 The Atlas App Services Command Line Interface ({+cli-bin+})
-allows you to programmatically manage your {+app+}s.
+allows you to programmatically manage your apps.
 With {+cli-bin+}, you can create or update
-{+app+}s from a local directory as well as export
+apps from a local directory as well as export
```

Comment:
all the `{+app+}` -> `app` should be capitalized as `App`

Reply:
this also in other files such as `source/cli/realm-cli-accessList-create.txt`. should be corrected across whole repo.